exporter/awsemfexporter/grouped_metric.go

```diff
@@ -18,10 +18,9 @@ import (
 	"encoding/json"
 	"strings"
 
+	aws "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics"
```

Comment:
This is not following the import grouping rules used by this repo and will fail the lint checks.  Revert the import changes to this file.

Reply:
Done too !! Thanks for your insight !!!